tests/vmi_multus_test.go

```diff
@@ -829,9 +829,10 @@ var _ = Describe("[Serial]SRIOV", func() {
 		// interfaces. It can be achieved either by configuring the external switch
 		// properly, or via in-PF switching for VFs (works for some NIC models)
 		pingThroughSriov := func(cidrA, cidrB string) {
```

Comment:
regarding commit `SRIOV tests, Add validation that network name is set in VMI scheme`:
```
... ethN naming which cant be depend on,
since it might be random each boot
```
how is ethN random? AFAIK it's always eth1 by default. is it any different in SRIOV?

Reply:
you cannot depend on ethN naming which is giving random order according the sequence it detects them,
its the problem Edy found recently